ops/model.py

```diff
@@ -2926,7 +2926,7 @@ def secret_info_get(self, *,
         args = []  # type: List[str]
         if id is not None:
             args.append(id)
-        if label is not None:
+        elif label is not None:  # elif because Juju secret-info-get doesn't allow id and label
```

Comment:
For the sake of not letting the edge case 'no id and no label passed' go through, why not refactor this as

```
if id:
 args.append(id)
elif not label:
 raise ValueError('pass ID or label')
else:
 args.append(...)
```

Reply:
Going to leave this as is for now -- that check is already done by the callers (`Model.get_secret` and `Secret.__init__`), and that's the pattern for most of these: the hook tool methods like `secret_info_get` don't generally do additional argumment checking.